src/aind_data_asset_indexer/codeocean_bucket_indexer.py

```diff
@@ -44,6 +47,169 @@ def __init__(self, job_settings: CodeOceanIndexBucketJobSettings):
         """Class constructor."""
         self.job_settings = job_settings
 
+    def _get_external_data_asset_records(self) -> Optional[List[dict]]:
+        """
+        Retrieves list of code ocean ids and locations for external data
+        assets. The timeout is set to 600 seconds.
+        Returns
+        -------
+        List[dict] | None
+          List items have shape {"id": str, "location": str}. If error occurs,
+          return None.
+        """
+        response = requests.get(
+            self.job_settings.temp_codeocean_endpoint,
+            timeout=600,
```

Comment:
Might need to catch the timeout exception so that the script doesn't error out

Reply:
added error handler